README.md

```diff
@@ -14,215 +14,225 @@
 |---|---|---|---|
 |Mode 1 - Full Open Weather Map|Low|No sensor input required, just an API key for Open Weather Map|Average of precipitation and evapotranspiration.|
 |Mode 2 - Full Sensor, but calculating evapotranspiration|High|Sensors are required for all inputs. All inputs are expected to be point-in-time, *except precipitation*. That sensor is normally provided by a weather station or weather service as a daily accumulative / 'total precipitation today' sensor and that is what is expected by the component|Most recent value for precipitation and average of evapotranspiration.
 |Mode 3 - Mixed|Medium|API key is required for any inputs that have not been provided sensors. All inputs are expected to be point-in-time, *except precipitation*. That sensor is normally provided by a weather station or weather service as a 'total precipitation today' sensor and that is what is expected by the component|If a sensor is provided for precipitation, most recent value for precipitation is used. Otherwise the average of both precipitation and evapotranspiration.|
 |Mode 4 -Not calculating|Very high|In this mode, just two sensors are required: one for precipitation and one for evapotranspiration. Both are expected to be daily accumulative sensors ('total today'), as is normally the case when provided by a weather service or weather station.|Most recent value of both precipitation and evapotranspiration.|
 
+
 ## Configuration
 
 In this section:
-- [One-time set up](#step-1-configuration-of-component)
-- [List of events, services entities and attributes created](#step-2-checking-entities)
-- [Example automation](#step-3-creating-automation)
-- [Optional settings](#step-4-configuring-optional-settings)
-### Step 1: configuration of component
-Install the custom component (preferably using HACS) and then use the Configuration --> Integrations pane to search for 'Smart Irrigation'.
+- [Installation](#step-1-installing-of-component)
+- [Configuration](#step-2-configuration-of-component)
+- [Services, Events, Entities and Attributes](#step-3-checking-services-events-and-entities)
+- [Example automation](#step-4-creating-automation)
+
+### Step 1: Installing of Component
+Install the custom component (preferably using HACS) and then use the Configuration --> Integrations pane to search for 'Smart Irrigation'. In your sidebar you will find a new entry for panel 'Smart Irrigation'
+- API Key for Open Weather Map (optional). Only required in mode 1) and 3). See [Getting Open Weater Map API Key](#getting-open-weather-map-api-key) below for instructions.
+
+
+### Step 2: Configuration of Component
 You will need to specify the following:
+
+#### GENERAL
+This page provides global settings.
+- Automatic duration calculation: If enabled set the time of calculation (HH:MM).
+- Automatic weather data update: If enabled specify how often sensor update should happen (minutes, hours, days). Warning: weatherdata update time on or after calculation time! 
```

Comment:
"weatherdata update time should not be on or after calculation time". Also bug #183 will change this - we will do a weather data update at calculation time if there is no weatherdata available yet.